doc/reference/networking/can_api.rst

```diff
@@ -0,0 +1,273 @@
+.. _can_interface:
+
+Controller Area Network (CAN)
+#############################
+
+.. contents::
+    :local:
+    :depth: 2
+
+Overview
+********
+
+Controller Area Network is a two-wire serial bus specified by the
+Bosch CAN Specification, Bosch CAN with Flexible Data-Rate specification and the
+ISO 11898-1:2003 standard.
+CAN is mostly known for its application in the automotive domain. However, it
+is also used in home and industrial automation and other products.
+
+A so-called CAN transceiver connects the two bus lines, CAN High (CAN H) and
+CAN Low (CAN L), from the CAN network to the CAN controller.
+The send wire from the controller to the transceiver is called CAN TX, and the
+receive wire is called CAN RX. These wires use the logic levels whereas the bus
+level is interpreted differential between CAN H and CAN L.
+Both lines, CAN H and CAN L, at the same level, is called the recessive state.
```

Comment:
This sentence sounds weird, not sure what we should say here.

Reply:
Rewritten